includes/query_processor.inc

```diff
@@ -232,6 +236,9 @@ class IslandoraSolrQueryProcessor {
       $params_date_facets["facet.date.start"] = 'NOW/YEAR-20YEARS';
       $params_date_facets["facet.date.end"] = 'NOW';
       $params_date_facets["facet.date.gap"] = '+1YEAR';
+      $params_date_facets["facet.range.start"] = 'NOW/YEAR-20YEARS';
+      $params_date_facets["facet.range.end"] = 'NOW';
+      $params_date_facets["facet.range.gap"] = '+1YEAR';
```

Comment:
Not sure these defaults are appropriate... do we have to do [some form of detection](https://github.com/Islandora/islandora_solr_search/pull/342/files#diff-2d69368f59ba81d5635cd8274dd66090R633), to determine if date-related parameters are appropriate? Also, brings up the question of what would be a reasonable default, otherwise, for numerical data and the like.